src/connection/host_lookup.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2008 Search Solution Corporation
```

Comment:
While looking for how to write a license for a newly created file, I found this line.
```suggestion

```

Reply:
Thank you! I fix it :)